next/components/common/HoverBoldButton.tsx

```diff
@@ -12,7 +12,7 @@ export default function HoverBoldButton({ text, dataLabel, onClick, className }:
             onClick={() => onClick()}
             data-label={dataLabel}
             aria-label={text}
-            className={`flex-grow bold-pseudo cursor-pointer px-3 py-2
+            className={`flex-grow  bold-pseudo cursor-pointer px-3 py-2
```

Comment:
Nitpick: is there a reason why a space was added here?

Reply:
> Nitpick: is there a reason why a space was added here?

I must have missed it when copying the styling, no specific reason. It does not impact functionality.

Reply:
Could you remove it anyways? Ideally this sort of thing is handled by automatic tools. At work we used prettier with tailwind to automatically run formatting on save to sort tailwind classes, snd a few other QoL things. Not saying we need that but its something you'd do in a work project.

Reply:
Yep, it's fixed